src/components/Cards/EventCard/EventCard.css

```diff
@@ -1,3 +1,8 @@
+.icon{
+  margin-top: 14px;
```

Comment:
You were told to make the component responsive. I asked you to use em, rem, etc.